v2/trace/aws/doc.go

```diff
@@ -0,0 +1,37 @@
+// Package awstrace contains aws session wrapper for tracing through SNS and SQS
```

Comment:
I quite like the idea of having examples in a separate package 
like put this in ./example/main.go and then not have is comments.

Then you get an compiler reminder to update docs when you are updating the code.

But maybe this is the go way?

Reply:
Good idea so the compiler catches all errors, but I still would like to have an short highlighted example in the doc.go

Reply:
Better now?